contracts/src/models/combat.cairo

```diff
@@ -0,0 +1,42 @@
+#[derive(Model, Copy, Drop, Serde)]
+struct Health {
+    #[key]
+    entity_id: u128,
+    value: u128
+}
+
+#[derive(Model, Copy, Drop, Serde)]
+struct Attack {
+    #[key]
+    entity_id: u128,
+    value: u128
+}
+
+
+#[derive(Model, Copy, Drop, Serde)]
+struct Defence {
```

Comment:
Given the amount of effort it required for Britain to colonise Nigeria, I do not think the Queen Elizabeth (God bless her soul) would have been happy to see me spell it as `Defense`. It would seem like all that effort had been in vain. So I hope you see the reason for my decision

Reply:
god bless the queen 🇬🇧 👑